assembly/src/assembler/instruction/mod.rs

```diff
@@ -40,6 +40,8 @@ impl Assembler {
         let result = match instruction {
             Instruction::Assert => span.add_op(Assert),
             Instruction::AssertEq => span.add_ops([Eq, Assert]),
+            Instruction::AssertEqw => span
+                .add_ops([MovUp4, Eq, Assert, MovUp3, Eq, Assert, MovUp2, Eq, Assert, Eq, Assert]),
```

Comment:
Did you consider putting this in `field_ops.rs` as a function? Most of the other "complex" assembly instructions are defined as a function with docs.  Might be something to consider what do you think?

Reply:
moved code to `field_ops`